### DISCUSSION TITLE
Renaming master branches

### DISCUSSION THREAD

**Discussion by nichollsh** [FormingWorlds/PROTEUS#156] [Ideas]
It is generally encouraged to use `main` as the name of the default branch of a repository, instead of `master` which has historically been the default. While AGNI already uses `main`, all of the FWL repositories currently use `master`. 

We should consider switching to `main` since it is offensive to some people and the change is relatively easy to perform.

See also: https://github.com/github/renaming and https://sfconservancy.org/news/2020/jun/23/gitbranchname/.

Thoughts?

@timlichtenberg @lsoucasse @stefsmeets @EmmaPostolec @planetmariana

**Comment by timlichtenberg** [comment]
Good idea. Fine for me if it can be performed without too much background noise.

**Reply by timlichtenberg** [reply]
Also, well done to finally starting a `Discussion` on here! :-)

**Reply by stefsmeets** [reply]
Muscle memory for main is strong for me, so looking forward to this change 😅

Make sure that everyone updates their github remote though (`git remote -v`)

**Comment by nichollsh** [comment]
Seems like there's interest in this idea, so I will make an issue.
